db/system_keyspace.cc

```diff
@@ -296,11 +296,6 @@ schema_ptr system_keyspace::cdc_generations_v3() {
              * range when the generation was first created. Together with the set of streams above it fully
```

Comment:
> In `topology_coordinator:clean_obsolete_cdc_generations`, we compute the iterator of the last obsolete generation (`last_obsolete_gen_it`). For now, it is unnecessary, but we will use it in the following patch to clean `committed_cdc_generations` from obsolete generations.

But you do use it already in this patch, to remove data from CDC_GENERATIONS_V3.

Reply:
In this patch, I only need this: `auto id_upper_bound = (*last_obsolete_gen_it)->id;`. I don't need the `last_obsolete_gen_it` variable. I can find `id_upper_bound` directly in the loop.

Anyway, it looks like writing this paragraph wasn't a good idea since it's confusing and not important. I'll remove it.